test/plugin-nested-docs/e2e.spec.ts

```diff
@@ -136,5 +136,18 @@ describe('Nested Docs Plugin', () => {
       // TODO: add back when error states are fixed
       // await expect(parentSlugInChild).toHaveValue('/parent-slug-draft')
     })
+
```

Comment:
I would have probably used an `int.spec`. They tend to run faster and are easier to debug.